ocs_ci/utility/utils.py

```diff
@@ -3298,3 +3298,35 @@ def get_client_version(client_binary_path):
         resp = exec_cmd(cmd)
         stdout = json.loads(resp.stdout.decode())
         return stdout["releaseClientVersion"]
+
+
+def add_chrony_to_ocp_deployment():
+    """
+    Create and Add necessary chrony resources
+
+    """
+    for role in ["master", "worker"]:
+        log.info(f"Creating and Adding Chrony file for {role}")
+        ocp_version = get_ocp_version()
+        ign_version = ""
+        with open(constants.CHRONY_TEMPLATE) as file_stream:
+            chrony_template_obj = yaml.safe_load(file_stream)
+        chrony_template_obj["metadata"]["labels"][
+            "machineconfiguration.openshift.io/role"
+        ] = role
+        chrony_template_obj["metadata"]["name"] = f"99-{role}-chrony-configuration"
+        if Version.coerce(ocp_version) < Version.coerce("4.6"):
+            ign_version = "2.2.0"
+        elif Version.coerce(ocp_version) == Version.coerce("4.6"):
+            ign_version = "3.1.0"
+        elif Version.coerce(ocp_version) >= Version.coerce("4.7"):
+            ign_version = "3.2.0"
```

Comment:
How we will maintain this ignition version for next OCP versions? Where we can find out which version we should set here?   Can we document it somewhere or at least add comment about it?

Reply:
As we discussed offline it will be good maybe to move it to OCP version config flies:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/ocp_version/ocp-4.7-config.yaml

we will need to cover ocp-x.y.yaml + ocp-x.y-ga.yaml files. Once we will do for new version we will see it has to be checked and changed if we will have comment there.

Reply:
Done